Makefile

```diff
@@ -57,7 +57,7 @@ ETCD_VERSION = v3.5.5
 # PROTOC_INC_GO_MODULES are Go modules which must be resolved and included
 # with `protoc` invocations
 PROTOC_INC_GO_MODULES = \
-	github.com/golang/protobuf \
+	google.golang.org/protobuf \
```

Comment:
this module has been renamed, and `make` was failing as `go list github.com/golang/protobuf` was returning nothing